scripts/module/emote.nut

```diff
@@ -1,222 +1,227 @@
 function onPlayerCommand(player, cmd, text) {
-  cmd = cmd.tolower()
+  cmd = cmd.tolower();
   if (cmd == "e") {
     if(!text) {
-      MessagePlayer("[#6666FF]Anims' list: [#FFFF81](/a) stop, watcha, lean, talk, wave, die^, fall^, fallback^, :o, apply, bu!, arg^, carwback, wback, elbow^, rollback, boss, crawling, rollinfront, falling, roll, beware, idk, look, dumb, aim^, drown, handsup, wut, fu, phone, sit, stand, sell, hm?, onfoot, db^, heal, lay, bored, riot, dance, drunk",player);
-      MessagePlayer("[#6666FF]READ ME -> The symbol ^ means the anim has more options. Example: /a dance, /a dance 2, /a dance 3.",player);
+      MessagePlayer("[#6666FF]Anims' list: [#FFFF81](/a) stop, watcha, lean, " +
+		    "talk, wave, die^, fall^, fallback^, :o, apply, bu!, arg^" +
+		    ", carwback, wback, elbow^, rollback, boss, crawling," +
+		    "rollinfront, falling, roll, beware, idk, look, dumb," +
+		    "aim^, drown, handsup, wut, fu, phone, sit, stand, sell," +
+		    "hm?, onfoot, db^, heal, lay, bored, riot, dance, drunk",
+		    player);
+      MessagePlayer("[#6666FF]READ ME -> The symbol ^ means the anim has more options." +
+		    "Example: /a dance, /a dance 2, /a dance 3.",player);
```

Comment:
Just do the same as for `teleporters`.

auto-generate help message and just keep a long list with group and animation index.

```
emotes <- {
  "stop": [0, 29],
  "...": [...],
}
```